examples/substrait-spark/gradlew

```diff
@@ -0,0 +1,249 @@
+#!/bin/sh
```

Comment:
Does this submodule need it's own Gradle wrapper, or can it share the top-level one?

Reply:
Good point..